include/dlaf/eigensolver/reduction_to_band/impl.h

```diff
@@ -320,9 +330,12 @@ void setupReflectorPanelV(bool has_head, const SubPanelView& panel_view, const S
     // the blocksize, leading to just using a part of A (first full nrefls columns)
     spec.size = {spec.size.rows(), std::min(nrefls, spec.size.cols())};
 
-    auto tile_v = v(i);
-    pika::dataflow(getHpExecutor<Backend::MC>(), std::move(setupV0), std::move(tile_v),
-                   splitTile(mat_a.read(i), spec));
+    const auto p = dlaf::internal::Policy<B>(thread_priority::high);
+    dlaf::internal::whenAllLift(ex::keep_future(splitTile(mat_a.read(i), spec)), v.readwrite_sender(i)) |
+        matrix::copy(p) | ex::start_detached();
+
+    dlaf::internal::whenAllLift(blas::Uplo::Upper, T(0), T(1), v.readwrite_sender(i)) | tile::laset(p) |
+        ex::start_detached();
```

Comment:
Here there are two separate tasks when it could theoretically be a single one.

@msimberg @rasolca do you have any suggestion? if not, do you want me to add a note/issue about this?

Reply:
I think this is ok as it is. But an issue or a note doesn't hurt. It can always be ignored in the future!

Reply:
note just added in the code 😉